### DISCUSSION TITLE
How to export all Freetube parameters ?

### DISCUSSION THREAD

**Discussion by STVGecko** [FreeTubeApp/FreeTube#5307] [Q&A]
I can export history, subscriptions and playlists, but I can't find the option to export all the settings (interface colours, default quality, etc.).

How to export all Freetube parameters please ?

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
See file location for settings https://docs.freetubeapp.io/usage/data-location/#notable-files

**Comment by 4yman-0** [comment]
>Windows:
>%APPDATA%/FreeTube

>Mac
>~/Library/Application Support/FreeTube/

>Linux
>~/.config/FreeTube

`%APPDATA%` is the `AppData` hidden folder in user folder (C:\Users\\*USERNAME*\AppData)
`~` is the home directory (/Users/*USERNAME* on MacOS or /home/*USERNAME* on Linux)

There you should find Database files `*.db`. These contain subscriptions, history, playlists, settings...
